lib/rules/selector-disallowed-list/__tests__/index.mjs

```diff
@@ -203,7 +203,7 @@ testRule({
 		},
 		{
 			code: '.foo > /*comment*/.bar {}',
-			message: messages.rejected('.foo > .bar'),
+			message: messages.rejected('.foo > /*comment*/.bar'),
```

Comment:
This is user facing, but this is a minor improvement that follows on previous work to fix the reported ranges.

See: https://github.com/stylelint/stylelint/blob/971b61e85564975e9b2741f7c286ea248e3b243e/.changeset/sixty-kings-join.md?plain=1#L5

Reply:
Remember to add `& [@romainmenke](https://github.com/romainmenke)`

Reply:
I don't really need the attribution unless there is a requirement to add these? ☺️